packages/augur-ui/src/modules/account/components/account-view.styles.less

```diff
@@ -136,3 +136,56 @@
     }
   }
 }
+
+:root[theme="BETTING"],
+:root[theme="SPORTS"] {
+  .AccountView {
+    > div {
+      border: @size-1 solid var(--color-outline);
+      border-radius: @size-6;
```

Comment:
I would use `@border-radius-rounded` here (and all other @size-6 border-radius definitions) instead of 6 incase that ever changes.

Reply:
Updated. Thank you.